src/app/modules/react-migration/react-migration.controller.ts

```diff
@@ -225,6 +225,11 @@ export const servePublicForm: ControllerHandler<
 }
 
 export const serveDefault: ControllerHandler = (req, res, next) => {
+  // Delete the deprecated cookie.
+  if (req.cookies) {
```

Comment:
We could check if there's the old cookie at all before sending the delete.

Reply:
[7ebfd3b](https://github.com/opengovsg/FormSG/pull/5665/commits/7ebfd3b5fead285a25667d4bdce15d019e1253a7)